packages/migrations/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@aragon/contract-helpers-migrations",
```

Comment:
It's generally good practice to keep the names prefixed the same in the repo, so they're easily rememberable and guessable.

I think going with `@aragon/contract-helpers-*` sounds good enough; things like `contract-helpers-deploy` and `contract-helpers-test` sound quite obvious.